sites/hashdev/src/_pages/blog/1_block-design.mdx

```diff
@@ -250,6 +254,6 @@ Since we live in a world where we don't have infinite screen space, it can be tr
 
 ## Working with designers
 
-At this point, some of you might be overwhelmed by the number of things you need to consider when designing blocks. Design is hard! There's a reason it's an entire specialised profession that takes decades to master.
+At this point, some of you might be overwhelmed by the number of things you need to consider when designing blocks. Design is hard! There's a reason it's a specialized profession that takes decades to master.
 
-If you're working on a block and you want it to be _really_ great, you should find a designer to come work on it with you.
+If you're working on a block and you want it to be _really_ great, you should find a designer to come work on it with you. There sadly aren't many designers involved in open source projects, often because of the time committment it takes to fully understand a project and what it needs. It isn't as easy to hop in and start contributing design work in the same way we can with code. Thankfully, block-sized projects are smaller in scope than most OS ventures. They also have an explict need for great UX and visual design. Which makes them the perfect side project to pitch to your designer friends.
```

Comment:
```suggestion
If you're working on a block and you want it to be _really_ great, you should find a designer to come work on it with you. There sadly aren't many designers involved in open source projects, often because of the time commitment it takes to fully understand a project and what it needs. It isn't as easy to hop in and start contributing design work in the same way we can with code. Thankfully, block-sized projects are smaller in scope than most OS ventures. They also have an explict need for great UX and visual design. Which makes them the perfect side project to pitch to your designer friends.
```